src/Rmath.jl

```diff
@@ -42,6 +42,16 @@ end
     export dwilcox,pwilcox,qwilcox,rwilcox # Wilcox's Rank Sum statistic (m, n)
     export ptukey, qtukey              # Studentized Range Distribution - p and q only
 
+function __init__()
+
+    unsafe_store!(cglobal((:unif_rand_ptr,libRmath),Ptr{Void}),
```

Comment:
yes, this requires the corresponding change to Rmath-julia (I should have explained that in the PR comment)